packages/homepage/content/articles/codesandbox-ci-embeds/index.md

```diff
@@ -0,0 +1,108 @@
+CodeSandbox CI is the latest way maintainers can use CodeSandbox to save time
+and improve experiences for users and contributors. But more than 4000 open
+source projects already use CodeSandbox embeds in their docs, blog posts, and
+websites—providing visitors with working examples that help explain how to use
+their creations.
+
+Embeds provide the ability to view a sandbox inline in your content, whether you
+want to show the code, the running app, or both at the same time. Our embeds use
+a specially designed version of CodeSandbox that's small in size but almost as
+powerful as the full thing, making for a great way to explain a coding concept
+or show off a demo.
+
+We want you to be able to use embeds anywhere you want. So based on feedback
+from our community, we've launched a new default look and feel for embeds that's
+minimal in design and looks great wherever you place it. With a subtle color
+palette and no branding, it doesn't take the focus away from your great content.
+
+Danny Ruchtie, Head of Design here at CodeSandbox, notes that with this release,
+_“we've focused on building something that would better blend in with its
+surroundings while maintaining all its powerful features. So there's less focus
+on the UI and more on the content. We also wanted to give users more control and
+improve the embed experience on mobile.”_
+
+See the difference for yourself:
+
+<iframe
+  src="https://codesandbox.io/embed/new?fontsize=14"
+  style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+  title="new"
+  allow="geolocation; microphone; camera; midi; vr; accelerometer; gyroscope; payment; ambient-light-sensor; encrypted-media; usb"
+  sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"
+></iframe>
+
+### The next big design trend?
+
+Ahead of the hipster crowd and the recent dark mode trend (we've had a default
+dark theme since we started!), we're pleased to introduce what's sure to be the
+next big thing in web design: light mode!
+
+Seriously though, while black's our go-to, we appreciate it doesn't always work
+for some site designs. So we've added a new white-based theme for embeds, which
+you can toggle on when creating an embed from the Share menu in the editor.
+
+The new design is already live, so check it out and take a look at
+[our docs](https://codesandbox.io/docs/embedding) for more info.
+
+### Get more done
+
+With embeds, and now CodeSandbox CI, we're giving open-source project
+maintainers new ways to reduce their support overhead and get more done.
+Available now, CodeSandbox CI makes it possible to test fixes without the need
+to clone, install, and check things locally. There’s more info about how to
+configure CodeSandbox CI in [our docs](https://u2edh.csb.app/docs.html), and you
+can get started by
+[installing the GitHub app](https://github.com/apps/codesandbox).
+
+## Thanks
+
+Thanks goes to Bogdan Luca, Siddharth Kshetrapal, and Danny Ruchtie whose hard
```

Comment:
Since we always added a link to people's Twitter account, maybe do it in this post too? 🤔

```md
Thanks goes to [Bogdan Luca](https://twitter.com/lucabogdan), [Siddharth Kshetrapal](https://twitter.com/siddharthkp), and [Danny Ruchtie](https://twitter.com/druchtie) whose hard
```